python/MooseDocs/extensions/MooseBibtex.py

```diff
@@ -26,6 +28,26 @@ class MooseBibtex(MooseCommonExtension, Preprocessor):
   def __init__(self, markdown_instance=None, **kwargs):
     MooseCommonExtension.__init__(self, **kwargs),
     Preprocessor.__init__(self, markdown_instance)
+    self.macro_files = kwargs['macro_files']
+
+  def getMacroBib(self, bibfile):
```

Comment:
This is only used once and inline with the parse_file command. I think you should rename this function to "parseBibtexFile" or something similar and move the "parse_file" command into this function along with the temporary clean up. Doing this will keep the main "run" function cleaner, I don't like that the temporary is created in one function and deleted in another.